iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergOutputCommitter.java

```diff
@@ -598,9 +605,10 @@ private void commitWrite(Table table, String branchName, Long snapshotId, long s
    * @param partitionPath The path of the compacted partition
    */
   private void commitCompaction(Table table, Long snapshotId, long startTime, FilesForCommit results,
-      String partitionPath) {
-    List<DataFile> existingDataFiles = IcebergCompactionUtil.getDataFiles(table, partitionPath);
-    List<DeleteFile> existingDeleteFiles = IcebergCompactionUtil.getDeleteFiles(table, partitionPath);
+      String partitionPath, long fileSizeThreshold) {
+    List<DataFile> existingDataFiles = IcebergCompactionUtil.getDataFiles(table, partitionPath, fileSizeThreshold);
+    List<DeleteFile> existingDeleteFiles = fileSizeThreshold == -1 ?
```

Comment:
we don't write deletes cause `-1` stands for MAJOR/FULL table/partition compaction?

Reply:
I removed this check, assuming that we want to compact all files with deletes regardless of file size.

Reply:
could you please add a comment for `-1`